### PR TITLE
AL new people import

### PR DESCRIPTION
**Pull request by chrisyamas**
Four manual changes that were made during import that are of note (all changes were made in consultation with @NewAgeAirbender):
- Wes Allen was moved to `/executive` given his election from lower chamber member to Secretary of State, with details updated accordingly
- Marcus Paramore, his lower chamber district seat successor, had a `start_date` manually added
- Merika Coleman, who was elected to upper while an incumbent lower chamber member, had old file updated to include new senate info, and other details updated accordingly regarding start_date, end_date, etc., making sure to keep her `id` the same as before
- Patrick Sellers, her lower chamber district seat successor, had a `start_date` manually added

### CODE REVIEW

**Comment by chrisyamas** [comment]
Working on lint fixes now...

File: data/al/legislature/Joe-Lovvorn-972dcfd0-f2a2-438b-b5de-276ceb398a91.yml [hunk 1]
```diff
@@ -12,17 +12,23 @@ roles:
   district: '79'
 offices:
 - classification: capitol
-  address: 11 South Union Street, Suite 435, Montgomery AL
+  address: 11 South Union Street,Suite 435,Montgomery, AL 36130-2950
   voice: 334-261-0540
+- classification: district
+  address: '515 Ogletree Road,Office: ,Auburn, AL 36830'
```
Comment (NewAgeAirbender) [inline data/al/legislature/Joe-Lovvorn-972dcfd0-f2a2-438b-b5de-276ceb398a91.yml:18]: Looks like some of the offices were added to the API weird cuz it's also wrong on the source. Need to remove `Office: ,`

File: data/al/legislature/Paul-W-Lee-37a5dfa8-e69f-4567-b8c0-3ea35a69f181.yml [hunk 2]
```diff
@@ -12,10 +12,15 @@ roles:
   district: '86'
 offices:
 - classification: capitol
-  address: 11 South Union Street, Suite 410 F, Montgomery AL
+  address: 11 South Union Street,Suite 410 F,Montgomery, AL 36130-2950
   voice: 334-261-0488
+- classification: district
+  address: '304 Ashborough Circle,Office: Suite 1,Dothan, AL 36301'
```
Comment (NewAgeAirbender) [inline data/al/legislature/Paul-W-Lee-37a5dfa8-e69f-4567-b8c0-3ea35a69f181.yml:18]: Here too, need to remove `Office: ,`

File: data/al/legislature/Ben-Robbins-7da5c799-bd67-458a-b3b1-a0cdda8f474e.yml [hunk 3]
```diff
@@ -12,15 +12,21 @@ roles:
   district: '33'
 offices:
 - classification: capitol
-  address: 11 South Union Street, Suite 526-A, Montgomery AL
-  voice: 334-261-0500
+  address: 11 South Union Street, Suite 526-A,Montgomery, AL 36130-2950
+  voice: 334-261-0477
+- classification: district
+  address: '29 West 3rd St,Office: Suite C, Sylacauga, AL 35150'
```
Comment (NewAgeAirbender) [inline data/al/legislature/Ben-Robbins-7da5c799-bd67-458a-b3b1-a0cdda8f474e.yml:18]: Same here

File: data/al/legislature/David-Faulkner-9b9fd235-1045-4935-a097-bd5e51d2c592.yml [hunk 4]
```diff
@@ -12,10 +12,15 @@ roles:
   district: '46'
 offices:
 - classification: capitol
-  address: 11 South Union Street, Suite 400-D, Montgomery AL
+  address: 11 South Union Street,Suite 400-D,Montgomery, AL 36130-2950
   voice: 334-261-0442
+- classification: district
+  address: '505 North 20th Street,Office: Suite 1800,Birmingham, AL 35203'
```
Comment (NewAgeAirbender) [inline data/al/legislature/David-Faulkner-9b9fd235-1045-4935-a097-bd5e51d2c592.yml:18]: & here

File: data/al/legislature/Frances-Holk-Jones-3bc2e263-71cf-41e2-80c5-3f97cad999ff.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+image: https://www.legislature.state.al.us/pdf/house/members/Holk-Jones_95.png
+party:
+- name: Republican
+roles:
+- type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:al/government
+  district: '95'
+offices:
+- classification: district
+  address: '315 E Laurel Ave,Office: Suite 101,Foley, AL 36535'
```
Comment (NewAgeAirbender) [inline data/al/legislature/Frances-Holk-Jones-3bc2e263-71cf-41e2-80c5-3f97cad999ff.yml:15]: & here

**Review by NewAgeAirbender — CHANGES_REQUESTED** [review]
Need few quick address changes to remove text & that its a string, but otherwise looks good

**Review by bfossen-ce — APPROVED** [review]
LGTM

**Review by NewAgeAirbender — APPROVED** [review]

**Pull request merged**